### PR TITLE
Make 'Use Max' flow consistent with browser extension

### LINKED ISSUE #1724: Send amount input does not accept commas from iOS numeric keypad
**Describe the bug**
Send amount input does not accept commas from iOS numeric keypad (invalid amount error)

**Screenshots**
Screen Shot 2020-07-27 at 7 58 42 PM

**To Reproduce**
_Steps to reproduce the behavior_

**Expected behavior**
Should either work with the comma, or convert commas into periods

**Smartphone (please complete the following information):**

-------------------------------------------------------------
_to be added after bug submission by internal support / PM_
**Severity**

- How critical is the impact of this bug on a user?
- Add stats if available on % of customers impacted
- Is this visible to all users?
- Is this tech debt?

### PR DESCRIPTION
**Pull request by EtDu**
**Description**

In the extension, max is a 'mode' that toggles on and off. When on, the input field becomes disabled and when the gas price changes the amount updates accordingly. On mobile the input remains enabled- and when the gas price changes it shows insufficient funds, which isn't an ideal user flow.

**Checklist**

**Issue**
https://trello.com/c/MI0kqA1l/186-make-use-max-send-flow-consistent-with-the-browser-extension

### CODE REVIEW

File: app/components/Views/SendFlow/Amount/index.js [hunk 1]
```diff
@@ -949,12 +995,13 @@ class Amount extends PureComponent {
...
 						<TextInput
 							ref={this.amountInput}
-							style={styles.textInput}
+							style={[styles.textInput, maxMode && styles.textInputMaxOn]}
 							value={inputValue}
 							onChangeText={this.onInputChange}
 							keyboardType={'numeric'}
 							placeholder={'0'}
 							testID={'txn-amount-input'}
+							editable={!maxMode}
```
Comment (estebanmino) [inline app/components/Views/SendFlow/Amount/index.js:1004]: is the user presses `use max` and then don't want to use that value, there's no way to update it without pressing `use max` again? that feels like something users won't like?

**Comment by estebanmino** [comment]
@EtDu @omnat do we want this?

**Comment by omnat** [comment]
let's not review/merge now, see my latest comment here: https://trello.com/c/MI0kqA1l/186-make-use-max-send-flow-consistent-with-the-browser-extension, waiting for Etienne's response.

**Comment by omnat** [comment]
Reviewing this new UX flow suggested by @EtDu now.

**Comment by EtDu** [comment]
As I mentioned on the trello card, I think it's a good idea. It provides a more straightforward/foolproof flow for the user. I'm open for any further discussion

**Comment by EtDu** [comment]
Closing this for now, we can reopen if we'd like to bring from backlog

**Pull request closed without merging**